jumpstarter/resources.py

```diff
@@ -89,6 +91,24 @@ def __get__(self, instance, owner):
             return instance._resources[self._name]
 
 
+class ThreadedContextManager(ObjectProxy):
+    def __init__(
+        self, context_manager: typing.ContextManager, capacity_limiter: CapacityLimiter
+    ):
+        super(ThreadedContextManager, self).__init__(context_manager)
```

Comment:
Since this is targeting newer versions of Python, why the verbosity?

Reply:
Oh right! That's how PyCharm autocompletes it, unfortunately.